index.html

```diff
@@ -26,24 +26,42 @@
       <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/fontawesome.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous"/>
 			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/regular.min.css" integrity="sha512-k2UAKyvfA7Xd/6FrOv5SG4Qr9h4p2oaeshXF99WO3zIpCsgTJ3YZELDK0gHdlJE5ls+Mbd5HL50b458z3meB/Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 		</head>
+    
 		<body>
+
 			<main class="main">
+        <nav class="mobileNavItem">   
+          
+            <ul id="menu" class="menu">
+              <div class="mobileNavItemu">
+                <li class=" mobile_portfolio_frame "><a href="#work" class="mobileNav_portfolio mobilefont">Portfolio</a></li>
+                <li class=" mobile_about_frame"><a href="#base-about-my-self"  class=" mobileNav_about mobilefont">About</a></li>
+                <li class="mobile_contact_frame"><a href="#contact-background"  class=" mobileNav_contact mobilefont">Contact</a></li>
+              </div>
+            </ul>
+         
+        </nav>
 				<header class="header-background-mobile">
           <nav class="AppBar-Compact-Dark-iOS">
             <div id="Toolbar-Dark-Compact">
               <div class="Normal-Buttn-Tert-Enabled">
-                <button id="NavIcon-Menu" ><div onclick="openNav()"><i class="fa-solid fa-bars fa-xl"></i></div></button>
+                <div  id="hamburger" class="close"  onclick="openNav()">
```

Comment:
- [ ] Please as requested by the first reviewer, kindly add a close button to close the hamburger menu.

### your design
![1](https://user-images.githubusercontent.com/94551484/211064271-8f773cba-dd64-425c-b34e-f23079e03852.png)


### figma design
<img width="229" alt="f1" src="https://user-images.githubusercontent.com/94551484/211064331-4e1d275b-f2ee-483c-811a-ed0ca3fed640.png">